tests/PHPUnit/Fixtures/OneVisitorTwoVisits.php

```diff
@@ -106,7 +106,9 @@ private function trackVisits()
 
         // Click on external link after 6 minutes (3rd action)
         $t->setForceVisitDateTime(Date::factory($dateTime)->addHour(0.1)->getDatetime());
-        self::checkResponse($t->doTrackAction('http://dev.piwik.org/svn', 'link'));
+
+        // Testing Outlink that contains a URL Fragment
+        self::checkResponse($t->doTrackAction('https://outlinks.org/#!outlink-with-fragment-<script>', 'link'));
```

Comment:
In addition to modifying a fixture (and thus testing archiving as well as tracking), would it be possible to add a test in System/TrackerTest.php? This test would test just the tracker behavior. Could add it to Integration/TrackerTest.php instead, though would be better to do this after tracker encapsulation.